tests/lib/tools/nested-state

```diff
@@ -0,0 +1,148 @@
+#!/bin/bash -e
+
+show_help() {
+    echo "usage: prepare"
+    echo "       restore"
+    echo "       build-image IMAGE-TYPE"
+    echo "       boot-image IMAGE-TYPE [--param-cd PARAM] [--param-mem PARAM]"
+    echo "       start-vm"
+    echo "       shutdown-vm"
+    echo "       delete-vm"
+    echo ""
+    echo "Available options:"
+    echo "  -h --help   show this help message."
+    echo ""
+    echo "COMMANDS:"
+    echo "  prepare:     creates all the directories needed to run a nested test"
+    echo "  restore:     deletes all the directories and data used by nested tests"
+    echo "  build-image: creates an image using ubuntu image tool"
+    echo "  boot-image:  first boot of the image which it remains running"
+    echo "  start-vm:    start the booted vm"
+    echo "  shutdown-vm: shutdown the running vm"
+    echo "  delete-vm:   delete the booten vm"
```

Comment:
```suggestion
    echo "  delete-vm:   delete the booted vm"
```

But also the "booted" word is weird, as again, that's a property of a system, not an image. The command name is good, I'd just say that it removes a system with a given name.

Reply:
New it is -> remove-vm:   removes a vm